docs/tutorials/getting-started/java/dropwizard.mdx

```diff
@@ -0,0 +1,315 @@
+import TabItem from '@theme/TabItem';
+
+# Getting Started with the OpenFeature Java SDK and Dropwizard
+
+## Introduction
+
+This walk-through teaches you the basics of using OpenFeature in Java in the context of a Dropwizard application.
+
+You'll learn how to:
+
+- Integrate the OpenFeature Java SDK
+- Install and configure the OpenFeature provider
+- Perform basic feature flagging
+
+## Requirements
+
+This walk-through assumes that:
+
+- You have a basic knowledge of Java and Dropwizard
+- You have Java 8 or later
+- You have Docker installed and running on the host system
+
+This guide uses Java 21 syntax, adjustments might be needed for earlier Java versions.
+
+## Walk-through
+
+### Step 1: Create a Dropwizard application
+
+Create a new Dropwizard application following the guide on [dropwizard.io](https://www.dropwizard.io/en/stable/getting-started.html).
+
+For this example we are using the maven archetype
+
+```shell
+$ mvn archetype:generate -DarchetypeGroupId=io.dropwizard.archetypes -DarchetypeArtifactId=java-simple -DarchetypeVersion=4.0.13
+...
+Define value for property 'name': OpenFeatureExample
+...
+Define value for property 'groupId': dev.openfeature
+Define value for property 'artifactId': openfeature-example
+Define value for property 'version' 1.0-SNAPSHOT:
+Define value for property 'package' dev.openfeature:
+Confirm properties configuration:
+name: OpenFeatureExample
+description: null
+shaded: true
+groupId: dev.openfeature
+artifactId: openfeature-example
+version: 1.0-SNAPSHOT
+package: dev.openfeature
+ Y: Y
+...
+```
+
+We now have an application we can open
+
+```shell
+cd openfeature-example
+```
+
+### Step 2: Add dependencies
+
+For dropwizard we can either use the SDK directly, or use a community module [dropwizard-openfeature](https://github.com/sideshowcoder/dropwizard-openfeature),
+and while dropwizard-openfeature provides some benefits like built-in healthchecks, and managing the startup and shutdown of resources associated with the
+OpenFeature Java SDK, it is not however not officially supported, so use at your own risk.
+
+Depending what you choose add the following to your `pom.xml`
+
+<Tabs groupId="dependency">
+<TabItem value="sdk" label="sdk-only">
+
+```xml
+<dependency>
+    <groupId>dev.openfeature</groupId>
+    <artifactId>sdk</artifactId>
+    <version>1.15.1</version>
+</dependency>
+```
+
+</TabItem>
+<TabItem value="dropwizard-module" label="dropwizard-openfeature">
+
+```xml
+<dependency>
+    <groupId>io.github.sideshowcoder</groupId>
+    <artifactId>dropwizard-openfeature</artifactId>
+    <version>1.0.0</version>
+</dependency>
+```
+
+</TabItem>
+</Tabs>
+
+Similar to the Spring Boot example we can use `flagd` as the provider, and would need to add the relevant provider as needed.
+In this example we are going to use the built-in `InMemoryProvider`, which would normally primarily used for testing purposes.
+
+### Step 3: Add code
+
+### Using the SDK only
```

Comment:
This section could also be split using tabs. If you use the same tag value, it will automatically be synced across the page.